src/ontology/cl-edit.owl

```diff
@@ -31101,6 +31104,38 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:36543915") Annotation(ob
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4040000 <https://orcid.org/0000-0002-0819-0473>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4040000 "2023-04-03T14:54:32Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:10719353") oboInOwl:hasBroadSynonym obo:CL_4040000 "glial restricted precursor")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:11921204") oboInOwl:hasExactSynonym obo:CL_4040000 "tripotential glial-restricted precursor cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:11756508") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4040000 "GRP cell")
+AnnotationAssertion(rdfs:label obo:CL_4040000 "glial restricted tripotential precursor cell")
+SubClassOf(obo:CL_4040000 obo:CL_0000123)
+SubClassOf(obo:CL_4040000 obo:CL_0011115)
+
+# Class: obo:CL_4040001 (chorionic girdle cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:27280409") Annotation(oboInOwl:hasDbXref "PMID:35328607") obo:IAO_0000115 obo:CL_4040001 "Horse-specific highly invasive trophoblast cell that invades the endometrium where it forms endometrial cups.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4040001 <https://orcid.org/0000-0002-0819-0473>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4040001 "2023-04-03T15:08:05Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:27280409") oboInOwl:note obo:CL_4040001 "Humans, mice and rats have a highly invasive (hemochorial) placentation. Horses have a non-invasive epitheliochorial placenta, except for the transient endometrial cup.")
+AnnotationAssertion(rdfs:label obo:CL_4040001 "chorionic girdle cell")
+SubClassOf(obo:CL_4040001 obo:CL_0000351)
+SubClassOf(obo:CL_4040001 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0003124))
+
+# Class: obo:CL_4040002 (enteroglial cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:16336493") Annotation(oboInOwl:hasDbXref "PMID:17483847") obo:IAO_0000115 obo:CL_4040002 "Glial cell that provides support to the enteric nervous system. It is involved in enteric neurotransmission, in maintaining the integrity of the mucosal barrier of the gut and serves as a link between the nervous and immune systems of the gut.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4040002 <https://orcid.org/0000-0002-0819-0473>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4040002 "2023-04-03T15:15:40Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25170211") oboInOwl:hasExactSynonym obo:CL_4040002 "enteric glial cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:11169131") oboInOwl:hasExactSynonym obo:CL_4040002 "enteroglial cell")
```

Comment:
This appears to be the same as the label.
Was this meant to be "enteric glia" with synonym type PLURAL?

Reply:
Do not see "enteric glia" added as synonym.

Do not see related term in PMID:1169131.

Reply:
@bvarner-ebi Sorry for these typing errors: PMID:11169131 (one of the "1" was missing in the ref.) indicates "enteric glial cells" (the "l" was missing in my comment). I added PMID:11169131 to the synonym "enteric glial cell". Let me know if I should create a plural synonym ("enteric glial cells") separately.

Reply:
Thanks for updating the PMID.
Adding "enteric glial cells" does not seem necessary. 
Adding "enteric glia" as a related, plural synonym can be considered (PMID:11169131).